ui/package.json

```diff
@@ -1,35 +1,35 @@
 {
   "name": "retroquest-ui",
   "version": "0.0.0",
-  "sasslintConfig": ".sasslintrc",
   "scripts": {
     "ng": "ng",
     "start": "ng serve --configuration local",
     "build": "ng build",
     "build-prod": "ng build --prod --build-optimizer --output-path=../api/src/main/resources/static",
+    "test": "yarn unit-watch",
     "unit": "jest",
     "unit-watch": "jest --watchAll",
     "unit-coverage": "jest --coverage=true",
     "lint": "ng lint",
```

Comment:
I recommend using the --cache setting to speed up linting

https://duncanleung.com/why-slow-plugin-typescript-eslint-performance-issues/

Reply:
On my 12 core processor linux box linting takes 30 seconds.  with --cache setting and no changes it only takes 1 second